setup.py

```diff
@@ -0,0 +1,28 @@
+import setuptools
+import os
+
+with open("README.md", "r") as readme:
+    long_description = readme.read()
+
+setuptools.setup(
+    name="pyrefinebio",
+    version=os.getenv("VERSION"),
+    author="CCDL",
+    author_email="",
```

Comment:
@cgreene @jaclyn-taroni @dvenprasad is there a better email for this than g3w4k4t5n3s7p7v8@alexslemonade.slack.com?

Reply:
What is the email address going to be _used for_ generally speaking?

Reply:
from https://packaging.python.org/tutorials/packaging-projects/:
> author and author_email are used to identify the author of the package.

I guess if someone is like "I used your package and my computer lit on fire!!!" they can email us there?

Reply:
I guess it seems fine for now. I will admit that I don't understand how you would reply from that email easily and, if that's true, it may not end up being sufficient.

Reply:
Mmm that's a good point... Should we just use ccdl@alexslemonade.org?

Reply:
👍